src/System.Private.CoreLib/shared/System/IO/PersistedFiles.Unix.cs

```diff
@@ -10,75 +10,6 @@ namespace System.IO
 {
     internal static partial class PersistedFiles
     {
-        private static string? s_userProductDirectory;
-
-        /// <summary>
-        /// Get the location of where to persist information for a particular aspect of the framework,
-        /// such as "cryptography".
-        /// </summary>
-        /// <param name="featureName">The directory name for the feature</param>
-        /// <returns>A path within the user's home directory for persisting data for the feature</returns>
-        internal static string GetUserFeatureDirectory(string featureName)
```

Comment:
Doesn't System.Security.Cryptography.X509Certificates in CoreFX use this?

Reply:
Could we move the corefx part to corefx as partial class of this type to make it more obvious?

Reply:
corefx uses everything in this file.

Reply:
undone the change